app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/FriendsList.kt

```diff
@@ -105,8 +104,24 @@ private fun FriendsCard(friend: UserModel, navToFriend: () -> Unit) {
           // Text information
           Column(modifier = Modifier.padding(8.dp)) {
             Text(text = friend.userName ?: "Name error", fontWeight = FontWeight.Bold)
-            Text(
-                text = "Bio no yet implemented, but will be here", overflow = TextOverflow.Ellipsis)
+            // little easteregg
+            if (friend.userName == "Jérémy Doffey" || friend.userName == "Thimphou") {

```

Comment:
Ok nice easter egg but maybe don't hardcode stuff like this for somebody to find later and might forget to remove.

Reply:
owo